src/app/options/design/design.component.html

```diff
@@ -76,6 +76,13 @@ <h3>{{'options.design.colors' | translate}}</h3>
       </div>
     </div>
   </div>
+
+  <!-- random preset theme toggle -->
+  <div class="input">
+    <label for="randTheme">{{'options.design.randTheme' | translate}}</label>
+    <options-toggle id="randTheme" name="randTheme" [(ngModel)]="settings.config.design.randTheme" (ngModelChange)="ga.field('design.randTheme', settings.config.design.randTheme);"></options-toggle>
```

Comment:
If random theme is on, we should disable the custom background/foreground fields as they won't be used anyway.

Reply:
I hid the entire background/foreground input fields in the design panel as well as the grid of preset themes since, like you said, they won't be used anyways.

Here is what the design panel looks like with random theme turned in: 

<img src="https://files.slack.com/files-pri/T02E7E44761-F02Q862J3N2/customoptionshidden.png?pub_secret=b1b04c6a23" alt="drawing" width="50%"/>